src/applications/vaos/project-cheetah/index.jsx

```diff
@@ -20,13 +22,28 @@ import ConfirmationPage from './components/ConfirmationPage';
 import { selectFeatureProjectCheetah } from '../redux/selectors';
 import ReceivedDoseScreenerPage from './components/ReceivedDoseScreenerPage';
 import ContactFacilitiesPage from './components/ContactFacilitiesPage';
+import { FETCH_STATUS } from '../utils/constants';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 import useManualScrollRestoration from '../hooks/useManualScrollRestoration';
 import useFormRedirectToStart from '../hooks/useFormRedirectToStart';
 import useFormUnsavedDataWarning from '../hooks/useFormUnsavedDataWarning';
 
-export function NewBookingSection({ featureProjectCheetah }) {
+export function NewBookingSection({
+  windowsStatus,
```

Comment:
Follow up to in person walkthrough: generally, these are named for the type of check that we're doing. In EC, we're checking for open windows of time. So we could change this variable name to match what you put in the state, `newBookingStatus`.

Reply:
Done.